CHANGES.md

```diff
@@ -12,6 +12,7 @@ Grammars:
 - added 3rd party Motoko grammar to SUPPORTED_LANGUAGES [rvanasa][]
 - added 3rd party Candid grammar to SUPPORTED_LANGUAGES [rvanasa][]
 - fix(haskell) Added support for characters [CrystalSplitter][]
+- enh(dart) Add `base`, `interface`, `sealed`, and `when` keywords [Sam Rawlins][]
 
 [AdamRaichu]: https://github.com/AdamRaichu
 [Ali Ukani]: https://github.com/ali
```

Comment:
Please also add yourself here, for the linking...

Reply:
I would but I'm already on line 1051

Reply:
You have to be In each release section, the file is split for releases.

Reply:
Ah I see. Fair enough, done!